src/components/PurchaseOrder/PO.js

```diff
@@ -87,7 +89,20 @@ const PO = ({
 
   const fetchOrder = useCallback(
     () => mutator.orderDetails.GET()
-      .then(setOrder)
+      .then(orderResp => {
```

Comment:
basically, the chain is working wrong now. You've changed the behavior of then and catch. Please keep it as before, hust don't use shorthand for catch

Reply:
It's still there